tests/ports/webassembly/promise_with_resolvers.mjs

```diff
@@ -0,0 +1,19 @@
+// Test polyfill of a method on a built-in.
+
```

Comment:
this reminded me I should explicitly put a license [in here](https://github.com/ungap/with-resolvers) 🤣

jokes a part, glad my poly helped testing this stuff as we do use it all over the place 👍

Reply:
We take licensing/copyright seriously.  Should I add anything here?

Reply:
those 3 LOC aren't anything special and the purpose of that polyfill is to *die* ASAP (once all browsers support it) so don't worry about it, I haven't invented anything and the pattern is pretty simple too, the only trick there is the usage of `var` (on purpose) but that's also just part of the specs.

We're good, and you have the author of that poly stating that in here 👍